### DISCUSSION TITLE
having some redirect links for confort

### DISCUSSION THREAD

**Discussion by dev-a-loper** [npm/feedback#721] [Website] [answered]
I am usually finding npm packages by typing the url 
for example I type : npmjs.com/express to see the readme for express
but most of the time I want to visit repository
I think its better to have this link ( npmjs.com/express/repo ) redirect to ( https://github.com/expressjs/express )
its really handy for me ..

there could be many more of these
like npmjs.com/express/deps  to see dependencies

**Comment by MylesBorins** [comment, marked as answer]
Thanks for the suggestion @ehsan2003.

I don't think it would make sense to have /:package/repo be a redirect. You can use the CLI command `npm repo package` to get a similar redirect easily though!

Regarding a `deps` link you can do https://www.npmjs.com/package/express?activeTab=dependents

At this time we are not looking to add complexity to the URL scheme of packages, there are unexpected implications that can arise. Thanks again for the suggestions
